### PR TITLE
Add new fields in API reference

### PR DESCRIPTION
**Pull request by Pierre-PO**
New fields have been added to the API.
This PR modify the API reference to describe all of them.
You can find them in [Notion](https://www.notion.so/processout/cc97f4869749498893553e94d3bdb25d?v=fcb642debe394170860107db71f4fc29&p=2beed1ea084b45bda6630afe835519fc)

### CODE REVIEW

File: source/refs/content/payments/_invoices.md.erb [hunk 1]
```diff
@@ -14,7 +14,10 @@ using the ProcessOut API. To start using invoices, [let's create one](#create-an
 <span class="console-font">**transaction**</span><br />*transaction_id*<br />*[Transaction](#transactions) expandable* | Transaction generated by the invoice
 <span class="console-font">**subscription**</span><br />*subscription_id*<br />*[Subscription](#subscriptions) expandable* | Subscription for which the invoice was automatically generated. Automatically set by ProcessOut
 <span class="console-font">**token**</span><br />*token_id*<br />*[Token](#tokens) expandable* | Token used (or to be used) to capture the payment. This token must belong to the customer linked to the invoice, if any
-<span class="console-font">**details**</span><br />*list of InvoiceDetails expandable* | Details about the invoice, in the form of a list of InvoiceDetail object- see below for object reference
+<span class="console-font">**details**</span><br />*list of InvoiceDetails* | Details about the invoice, in the form of a list of InvoiceDetail object- see below for object reference
+<span class="console-font">**risk**</span><br />*list of InvoiceRisk* | Risk about the invoice, in the form of a list of Risk object- see below for object reference
```
Comment (manuel-huez) [inline source/refs/content/payments/_invoices.md.erb]: Is that a list or just an object?

File: source/refs/content/payments/_invoices.md.erb [hunk 2]
```diff
@@ -14,7 +14,10 @@ using the ProcessOut API. To start using invoices, [let's create one](#create-an
...
+<span class="console-font">**device**</span><br />*list of InvoiceDevice* | Device information about the invoice, in the form of a list of Device object- see below for object reference
```
Comment (manuel-huez) [inline source/refs/content/payments/_invoices.md.erb]: Is that a list or just an object?

File: source/refs/content/payments/_invoices.md.erb [hunk 3]
```diff
@@ -14,7 +14,10 @@ using the ProcessOut API. To start using invoices, [let's create one](#create-an
...
+<span class="console-font">**shipping**</span><br />*list of InvoiceShipping* | Shipping information about the invoice, in the form of a list of Shipping object- see below for object reference
```
Comment (manuel-huez) [inline source/refs/content/payments/_invoices.md.erb]: Is that a list or just an object?

File: source/refs/content/payments/_invoices.md.erb (hunk 1 above)
Comment (Pierre-PO) [reply source/refs/content/payments/_invoices.md.erb]: Bad copy paste. Fixed

**Pull request merged**